### PR TITLE
FEC-11998 Fixed readme

### PR DESCRIPTION
**Pull request by GouravSna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,211 @@
...
+                .setStreamType(CAFCastBuilder.StreamType.VOD)
+var mediaInfo: MediaInfo = castBuilder.build()
+
+```
+
+###### GoogleCast for Non Kaltura Customers
+
+```Kotlin
+
+val castBuilder = KalturaBasicCAFCastBuilder(playbackParams)
```
Comment (GouravSna) [inline README.md:133]: how to create playbackParams

File: README.md [hunk 2]
```diff
@@ -3,4 +3,211 @@
...
+
+We provide Cast Application Framework(CAF) for it. Our plugin is devloped on top of Google's designed CAF framework.
+
+Before understanding that how to setup Google cast plugin on Android, let's understand few terms,
+
+##### Sender Application: 
+Your Android app which gives the user the ability to connect to the casting device, is a Sender Application.
+
+##### Receiver Application:
+When your sender app connects to the cast device then cast device downloads an script from the server which is actually configured against the CastId for your application. Basically it's a Web reciever Application. [More Info](https://developers.google.com/cast/docs/web_receiver)
```
Comment (GouravSna) [inline README.md]: change the more info link to our receiver code

File: README.md (hunk 1 above)
Comment (GouravSna) [reply README.md:133]: done

File: README.md (hunk 2 above)
Comment (GouravSna) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -3,4 +3,240 @@
...
+        var playbackParams = PlaybackParams();
+        playbackParams.poster = "Poster_URL"
+        playbackParams.id = "ID"
+        playbackParams.duration = Duration
+        playbackParams.type = "Vod"
+        playbackParams.dvr = false
+        playbackParams.vr = null
+        playbackParams.dvr = false
+        playbackParams.progressive = listOf<Progressive>()
+        playbackParams.hls = listOf<Hls>()
```
Comment (GouravSna) [inline README.md]: Make it clear, why this is required and why setter is required

Comment (GouravSna) [reply README.md]: done

**Review by giladna — APPROVED** [review]

**Pull request merged**